terraform/Makefile

```diff
@@ -9,6 +9,10 @@ ifndef PR_NUMBER
 	$(error PR_NUMBER is undefined)
 endif
 
+.PHONY: generateMatrices
+generateMatrices:
+	$(MAKE) -C ../versions generateMatrices
```

Comment:
The `generateMatrices`  target is now executed prior to executing terraform apply/destroy, because it generates the matrices for linuxAndWindows and Suse. The suse matrix is currently used by the `ec2-suse-builders` project to spin up the SUSE EC2 instances.